backend/api/Database/Models/MissionTask.cs

```diff
@@ -40,6 +41,32 @@ public MissionTask(CustomTaskQuery taskQuery)
             RobotPose = taskQuery.RobotPose;
             TaskOrder = taskQuery.TaskOrder;
             Status = TaskStatus.NotStarted;
+            Type = "inspection";
+        }
+
+        public MissionTask(Pose robotPose, string type)
+        {
+            switch (type)
```

Comment:
What happens if the type provided is inspection?

Reply:
Changed to throwing an exception if the provided type is not Localization or DriveTo